aten/src/ATen/cpu/vec/vec_half.h

```diff
@@ -0,0 +1,59 @@
+#include <ATen/cpu/vec/intrinsics.h>
+
+#ifdef CPU_CAPABILITY_AVX512
```

Comment:
why can't we just use `__at_align__` from `./aten/src/ATen/cpu/vec/vec_base.h`

Reply:
I can't directly move the codes of vec_half.h to vec_base.h as vec_base.h include half.h.
It will get compilation errors when them to vec_base.h.